js/stdlib/ts/ETH_like.ts

```diff
@@ -266,6 +266,84 @@ const stepTime = async (): Promise<TransactionReceipt> => {
   return await transfer(faucet, acc, parseCurrency(0));
 };
 
+// ****************************************************************************
+// Event Cache
```

Comment:
I'm not going to read this closely. I assume it has the same logic as the ALGO above, so please revise and/or confirm it doesn't then I'll make more comments.